bundles/org.openhab.binding.pulseaudio/pom.xml

```diff
@@ -14,4 +14,25 @@
 
   <name>openHAB Add-ons :: Bundles :: Pulseaudio Binding</name>
 
+  <dependencies>
+    <dependency>
+      <groupId>com.googlecode.soundlibs</groupId>
+      <artifactId>mp3spi</artifactId>
+      <version>1.9.5.4</version>
+      <scope>compile</scope>
```

Comment:
I'm not quite sure, yet. But I think you can set the scope of all three to `provided`. Then, the dependencies are not included in the JAR of this binding, but are loaded by the framework via the `features.xml` at runtime. That would reduce the size of the addons distribution.

Reply:
I tried, but I got "BundleException: Could not resolve module [...]" when deploying (in a dev env or even in a full installation).
The developer documentation "Build System" seems to also advice the "compile" scope ?

Reply:
Strange, should work in general. However, I just noticed that the Build System section states that `compile` should be used if the dependency is only used in one binding, which seems to be the case here. If you use the `compile` scope, the entries in the features.xml can be removed.